lib/job-simulation.rb

```diff
@@ -1,17 +1,48 @@
-require './Stack.rb'
-require './Queue.rb'
+require_relative  './Stack.rb'
+require_relative  './Queue.rb'
 
 class JobSimulation
   attr_reader :workers, :waiting, :roll
 
   def initialize (jobs_available, job_seekers)
-
+    @jobs_available = jobs_available
+    @job_seekers = job_seekers
```

Comment:
Why did you save these two passed variables into instance vars? We'd only need to do that if we're going to use the variable values later, which we shouldn't need to.